lib/v1/gadgets/blogger.md

```diff
@@ -0,0 +1,12 @@
+gratipay.xml is a an example of Gadget for Blogger. To add it, go to
+you blog overview, then:
+
+    Layout -> Add a Gadget -> Add your own
+
+Enter URL ('https://' prefix didn't work for me):
+
+    grtp.co/v1/gadgets/blogger.xml
+
+Set your "Name on Gratipay" (widget type if you want) and view how
```

Comment:
But since we're using username everywhere else so far I guess we shouldn't block on that here.

Noted in the v2 meta-ticket: https://github.com/gratipay/inside.gratipay.com/issues/125.